<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>118</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>notification</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>78</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Detection</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>108</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Videoslice</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>61</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>78</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
